catalogs/catalog-hadoop/src/main/java/org/apache/gravitino/catalog/hadoop/HadoopCatalogOperations.java

```diff
@@ -771,6 +774,23 @@ FileSystem getFileSystem(Path path, Map<String, String> config) throws IOExcepti
               scheme, path, fileSystemProvidersMap.keySet(), fileSystemProvidersMap.values()));
     }
 
-    return provider.getFileSystem(path, config);
+    return provider.getFileSystem(path, convertGravitinoToHadoopConf(config));
+  }
+
+  private Map<String, String> convertGravitinoToHadoopConf(Map<String, String> gravitinoConf) {
+    Map<String, String> hadoopConf = Maps.newHashMap();
+
+    gravitinoConf.forEach(
+        (k, v) -> {
+          if (HadoopS3FileSystemConfig.GRAVITINO_KEY_TO_S3_HADOOP_KEY.containsKey(k)) {
+            hadoopConf.put(HadoopS3FileSystemConfig.GRAVITINO_KEY_TO_S3_HADOOP_KEY.get(k), v);
+          } else if (HadoopGCSFileSystemConfig.GRAVITINO_KEY_TO_GCS_HADOOP_KEY.containsKey(k)) {
+            hadoopConf.put(HadoopGCSFileSystemConfig.GRAVITINO_KEY_TO_GCS_HADOOP_KEY.get(k), v);
+          } else
+            hadoopConf.put(
+                HadoopOSSFileSystemConfig.GRAVITINO_KEY_TO_OSS_HADOOP_KEY.getOrDefault(k, k), v);
+        });
+
+    return hadoopConf;
```

Comment:
I think we can split and do this in each provider, we don't have to expose here with `if...else if...else`.

Reply:
Yeah, I had hesitated about this method and finally decided to make the code like the current one.  I will modify it.

Reply:
fixed